src/ContractInteract/OSTPrime.js

```diff
@@ -43,6 +60,121 @@ class OSTPrime {
     this._wrapRawTx = this._wrapRawTx.bind(this);
     this.unwrap = this.unwrap.bind(this);
     this._unwrapRawTx = this._unwrapRawTx.bind(this);
+    this.setCoGateway = this.setCoGateway.bind(this);
+    this.setCoGatewayRawTx = this.setCoGatewayRawTx.bind(this);
+    this.initialize = this.initialize.bind(this);
+    this.initializeRawTx = this.initializeRawTx.bind(this);
+  }
+
+  /**
+   * Setup for OSTPrime contract. Deploys the contract and initializes
+   * it. See {@link OSTPrime#initialize}.
+   *
+   * @param {Web3} web3 Web3 object.
+   * @param {OSTPrimeSetupConfig} config OSTPrime setup configuration.
+   * @param {Object} txOptions Transaction options.
+   *
+   * @param {Promise<OSTPrime>} Promise containing the OSTPrime instance that
+   *                           has been set up.
+   */
+  static setup(web3, config, txOptions) {
+    if (!config.valueToken) {
+      throw new Error(
+        'Mandatory configuration "valueToken" missing. Provide EIP20 token contract address.',
+      );
+    }
+
+    if (!config.organization) {
+      throw new Error(
+        'Mandatory configuration "organization" missing. Set config.organization address.',
+      );
+    }
+
+    OSTPrime.validateSetupConfig(config);
+
+    const deployParams = Object.assign({}, txOptions);
+    deployParams.from = config.deployer;
+    deployParams.gasPrice = 0;
```

Comment:
why?

Reply:
See my other comment about the same function. AFAIK this is supposed to prevent leakage of any base tokens of the auxiliary chain during initialization.

Reply:
I'll add a comment regarding why OSTPrime is initialized like this.